### PR TITLE
Rec next articles

### PR DESCRIPTION
**Pull request by Andazu**
## Description

As a user, I want to see recommended articles under an article I have read
  
## Changes

Main changes:
- Articles are shown in a carousel wheel below the article screen

Additional changes
- Navigating to the article is done by pressing it
- Transition is done by scrolling to the top of the screen
- A swipe up tutorial is rendered on the first card and removed smoothly when a swipe up gesture is made
- The font size scales according to the amount of characters in the title to ensure it fits

## Related Issues

- The articles shown are just passed from the article screen and not actually recommended articles based on any user behavior
- Font scaling needs to be tested more with other titles
- This component might have made the scrollbar at the top of the screen less smooth?

## Checklist

- [x] Code has been tested locally and passes all relevant tests.
- [x] Documentation has been updated to reflect the changes, if applicable.
- [x] Code follows the established coding style and guidelines of the project.
- [x] All new and existing tests related to the changes have passed.  (on test fails because the test is broken, the function works as intended. The test have been ignored) 
- [x] Any necessary dependencies or new packages have been properly documented.
- [x] Pull request title and description are clear and descriptive.
- [x] Reviewers have been assigned to the pull request.
- [x] Performance impact of the changes has been evaluated, if relevant.

## Screenshots (if applicable)

441209354_1377430539596434_3209076793941844964_n

441153047_2091463227893848_7005394926055807328_n

## If mobile/frontend pull request, what version of the backend is it stable, and running on? 

dev

## Notes for Reviewers

<3

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -25,7 +25,7 @@
     "react-native": "0.73.4",
     "react-native-safe-area-context": "4.8.2",
     "react-native-screens": "~3.29.0",
-    "supabase": "^1.153.4"
```
Comment (benceisonline) [inline package.json:28]: we should probs keep supabase

File: screens/Article.jsx [hunk 2]
```diff
@@ -32,9 +33,6 @@ export default function Article({ route }) {
 		width: `${scrollPercentage}%`,
 	};
 
-	useEffect(() => {
-	}, []);
-
```
Comment (benceisonline) [inline screens/Article.jsx:37]: are we 100% sure that removing this is on purpose

File: components/NewsCard.jsx [hunk 3]
```diff
@@ -7,7 +7,7 @@ import PropTypes from 'prop-types';
 
 const { height } = Dimensions.get('window');
 
-export default function NewsCard({ article, userID, scrollPercentage, articlesInView}) {
+export default function NewsCard({ article, scrollPercentage, articlesInView}) {
```
Comment (benceisonline) [inline components/NewsCard.jsx:10]: what was userID used for before

File: package.json (hunk 1 above)
Comment (Andazu) [reply package.json:28]: Fixed

File: screens/Article.jsx (hunk 2 above)
Comment (Andazu) [reply screens/Article.jsx:37]: Added it back, no changes, will keep it added tho

File: components/NewsCard.jsx (hunk 3 above)
Comment (Andazu) [reply components/NewsCard.jsx:10]: it is not used in the code anywhere of that file, so might as well keep it deleted

**Review by benceisonline — APPROVED** [review]
lgtm

**Pull request merged**